### ISSUE TITLE
Feature Request: Exploding Downward

### ISSUE THREAD

**Issue by seraphim72** [valentine195/obsidian-dice-roller#84]
It would be nice to be able to have dice that explode downward.  It would re-roll and then subtract the resultant roll. 

For example on a roll of 1 on a 1d6, it would roll again (say a 5 this time) then subtract 1 + -5  = -4. It could then explode for another roll based on some formula, say in this case on a roll of six. So it might look like 1 + -6 = -5 + -{whatever is rolled next, and if another six it continues}

The short version is that I am asking for a way to roll dice for Rolemaster / Middle Earth Role Playing / High Adventure Role Playing and other d100 systems.  To do these in particular they would need to set some variable based on the initial roll. If it were a high positive it would continuously add and if it were a low negative it would continuously subtract. 

Thanks for the dice roller and the consideration.

**Comment by sigrunixia** [comment]
Hi @seraphim72 

Can you confirm if the later [addition and updates](https://github.com/javalent/dice-roller#conditions) to conditions and modified conditions solved your use case?
